docs/deployment/pull-backup.rst

```diff
@@ -99,17 +105,33 @@ stuff and clean up:
     umount /tmp/sshfs
     rmdir /tmp/sshfs
 
-Thanks to secuser on IRC for this howto.
+Thanks to secuser on IRC for this how-to!
+
+Full restore
+~~~~~~~~~~~~
+
+In a full restore differences between backup and client do not matter, so there
```

Comment:
Don't say what is not the case, say what is the case, like:

When doing a full restore, you restore all files (including the ones containing the ID-to-name mapping, /etc/passwd and /etc/group). So everything will be consistent automatically if you restore the numeric IDs stored in the archive.